test/recurrent/rnn-time-step.js

```diff
@@ -116,7 +116,7 @@ describe('RNNTimeStep', () => {
   it('can learn more than one to predict forwards and backwards', () => {
     const net = new LSTMTimeStep({
       inputSize: 2,
-      hiddenLayers: [20, 5, 1],
+      hiddenLayers: [10],
```

Comment:
Not sure how this file ended up in the PR.

Reply:
I meant to get it in sometime earlier. It simplifies testing.  I'll keep it to cut down on work resolving it.